examples/host_func_test.go

```diff
@@ -25,7 +25,7 @@ func Test_hostFunc(t *testing.T) {
 
 	// Host-side implementation of get_random_string on Wasm import.
 	getRandomString := func(ctx *wasm.HostFunctionCallContext, retBufPtr uint32, retBufSize uint32) {
-		const bufferSize = 10
+		const bufferSize = 10000
```

Comment:
This makes it necessary for Wasm program to grow memory space, and the growing memory fails when  we don't execute _start function beforehand. That's it!

Reply:
ok so this fails nicer in missing `_start` in other words, right?

```suggestion
		const bufferSize = 10000 // force memory space grow to ensure eager failures on missing setup 
```